### ISSUE TITLE
Add an installation manual

### ISSUE THREAD

**Issue by athityakumar** [intuit/judo#11]
Hey there! 👋 

Judo looks like a pretty interesting CLI tool that could probably replace language-specific CLI unit-tests with a centralized YAML-based testing.

The existing CONTRIBUTING.md and README.md don't include the installation manual yet - it'd really make it easier for new contributors to get started with developing Judo locally. 😅

**Comment by athityakumar** [comment]
I think this is the flow:

```sh
git clone https://github.com/intuit/judo
cd judo
npm install
npm run-script build
npm link
judo testing_file.yml
```

Let me know where you'd like to see the installation manual coming up: README.md, CONTRIBUTING.md or elsewhere. I'll submit a PR based on your response. 😄

**Comment by ejfrancis** [comment]
Hey @athityakumar 

Judo is package and distributed using npm, the package manager for node.js. You can see the installation command in the README as `npm i @intuit/judo`. But I think if someone isn't familiar with node.js this might not be totally obvious so we could definitely make this more clear in the README. Maybe something like this in the very beginning of the "Installation" section: 


"Judo is distributed and installed using [npm](https://www.npmjs.com), the package manager that ccomes bundled with [node.js](https://nodejs.org/en/). In order to be able to install Just you will need to install node.js first. Then you can run the following command:"

I think that would make it more clear, what do you think @athityakumar?

**Comment by ejfrancis** [comment]
As for local development your instructions are spot on, you'd need to `npm i`, then `npm run build` and then `npm link`

**Comment by athityakumar** [comment]
That sounds great, @ejfrancis! I've opened PR #14 to fix #11 😄
